README.rst

```diff
@@ -59,6 +59,42 @@ Dependencies
 - Some examples require external dependencies such as pandas. See the
   instructions in the `Building the docs`_ section for details.
 
+Examples
+========
+
+**Transparent and fast disk-caching of output value:** a memoize or make-like
+functionality for Python functions that works well for arbitrary Python objects,
+including very large numpy arrays. Separate persistence and flow-execution 
+logic from domain logic or algorithmic code by writing the operations as a set 
+of steps with well-defined inputs and outputs: Python functions. Joblib can 
+save their computation to disk and rerun it only if necessary:
+
+.. highlight:: python
+>>> from joblib import Memory
```

Comment:
The doc CI is red because this line is causing sphinx to complain:

```
/home/circleci/project/doc/README.rst:73:Explicit markup ends without a blank line; unexpected unindent.
```

The error message is not very explicit but I guess this means that the code snippet needs to be independent which makes sense in retrospect.